packages/wallet-adapter-react/src/utils.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  AptosStandardSupportedWallet,
+  Wallet,
+  WalletReadyState,
+  isRedirectable,
+} from "@aptos-labs/wallet-adapter-core";
+
+export type AptosWallet = Wallet | AptosStandardSupportedWallet<string>;
```

Comment:
that could live on the core package? and then any package can import it

Reply:
That's a good idea, I'll move it there

Reply:
I just realized that `@aptos-labs/wallet-standard` already exports an interface called `AptosWallet`. Any idea what we should call this instead?

Reply:
That's good enough for me. I added that type to wallet-core